CS483/CS483/Kartaclysm/Components/Abilities/ComponentAbilityConditions.cpp

```diff
@@ -116,6 +118,10 @@ namespace Kartaclysm
 		{
 			return false;
 		}
+		if (m_pRacer != nullptr && m_pRacer->HasFinishedRace())
```

Comment:
Rather than checking it every time, it would be better to handle it once as an assert() when it's assigned. The system should break loudly if it can't find the Racer Component.